### ISSUE TITLE
af

### ISSUE THREAD

**Issue by gshorer97** [emilioziniades/flatland#15]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by gshorer97** [comment]
Currently: One has to claim a square and change its colour at the same time with the added option to edit the colour later.

Instead: I suggest we start with the claim exclusively (i.e. keep the name and keep the colour), the user then has the same options of managing their owned squares

**Comment by gshorer97** [comment]
We could also have that square change to the selected state on the canvas once claimed so its easier to change colour and name as soon as it it claimed

**Comment by emilioziniades** [comment]
Agreed, this decoupling makes sense. To clarify, do you mean that when you claim a square, you don't choose a colour, and that colour changes always happen in the squareManager part of the app?

**Comment by gshorer97** [comment]
Yes, I think the user shouldn't need to change colour (or even enter the hex of the current colour) in order to claim that square.

Colour changes could happen in 2 locations within the app:
1) Within the squareManager 
2) Within the sidePanel IFF that particular square which is highlighted is owned by the user
